stac-spatio-temporal-asset-catalog/handout.md

```diff
@@ -1,53 +1,84 @@
-# Was ist der STAC? (TIM)
+# Was ist der STAC? (SpatioTemporal Asset Catalog)
```

Comment:
Hier fände ich es gut, wenn noch deutlicher wird, dass STAC versucht, den OGC Standard besser an Geodaten anzupassen.

Reply:
Habe jetzt in 0f54a69423dee35ba17efe749895cb5193856f44 einfach nur einen Stichpunkt, der das erwähnt, ergänzt und keine tiefergehenden Punkte aufgelistet...
Reicht dir das so oder hast du noch mehr im Sinn was man dazu schreiben könnte/sollte/müsste?